src/main/java/io/confluent/kafkarest/entities/TopicPartitionOffsetMetadata.java

```diff
@@ -97,7 +97,7 @@ public boolean equals(Object o) {
     if (offset != that.offset) {
       return false;
     }
-    if (metadata != that.metadata) {
```

Comment:
Yeah, this should be fine. tbh I'm not sure how this ended up incorrect, generally these are auto-generated by IntelliJ